src/components/Dashboard/scenes/Emails/EmailHistory.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Loading from '../../../../assets/svg/Loading';
+import { useEmails } from '../../../../hooks/useEmails';
+
+function truncateSubject(subject: string, wordLimit: number) {
+  const words = subject.split(' ');
+  if (words.length > wordLimit) {
+    return words.slice(0, wordLimit).join(' ') + '...';
+  }
+  return subject;
+}
```

Comment:
You don't need to use a function for this. Just use the truncate class https://tailwindcss.com/docs/text-overflow